components/eamxx/src/physics/rrtmgp/eamxx_rrtmgp_process_interface.cpp

```diff
@@ -105,9 +108,13 @@ void RRTMGPRadiation::set_grids(const std::shared_ptr<const GridsManager> grids_
   // Set of required gas concentration fields
   for (auto& it : m_gas_names) {
     // Add gas VOLUME mixing ratios (moles of gas / moles of air; what actually gets input to RRTMGP)
-    if (it == "o3") {
-      // o3 is read from file, or computed by chemistry
-      add_field<Required>(it + "_volume_mix_ratio", scalar3d_layout_mid, molmol, grid_name);
+    if (it == "o3" && !m_transient_ozone) {
+      if (!m_transient_ozone) {
```

Comment:
It looks like you have redundant logicals here...`if(!m_transient_ozone)` is inside the `if (it == "o3" && !m_transient_ozone)` block.